cirq/google/optimize.py

```diff
@@ -31,24 +31,24 @@ def _merge_rots(c: circuits.Circuit):
 
 
 _OPTIMIZERS = [
-    convert_to_xmon_gates.ConvertToXmonGates().optimize_circuit,
+    convert_to_xmon_gates.ConvertToXmonGates(),
```

Comment:
You can make mypy happy by adding `.optimize_circuit` to the end of each of these. Keeping as is is also okay.

Reply:
```
 for post in followup_optimizations:
        post(before).optimize_circuit
        post(expected).optimize_circuit
```
Gives:
`AttributeError: 'NoneType' object has no attribute 'optimize_circuit'`

Removing `.optimize_circuit` works fine but mypy complains that the object is not callable.

Going with the original 
```
 for post in followup_optimizations:
        post.optimize_circuit(before)
        post.optimize_circuit(expected)
```
also works to but mypy complains with a ` "object" has no attribute "optimize_circuit"`

Reply:
You want

```
 for post in followup_optimizations:
        post(before)
        post(expected)
```

Reply:
That's what i have right now.